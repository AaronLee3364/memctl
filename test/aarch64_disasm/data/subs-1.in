fffffff0070c0ac0 f10400a5
