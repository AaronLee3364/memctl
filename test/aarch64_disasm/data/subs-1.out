fffffff0070c0ac0  SUBS    X5, X5, #0x100
